Sources/Storyboard/StoryboardBased.swift

```diff
@@ -28,7 +28,7 @@ public protocol StoryboardBased: class {
 public extension StoryboardBased {
   /// By default, use the storybaord with the same name as the class
   static var storyboard: UIStoryboard {
-    return UIStoryboard(name: String(self), bundle: NSBundle(forClass: self))
+    return UIStoryboard(name: String(describing: self), bundle: Bundle(for: self))
```

Comment:
Looks like `StoryboardBased` doesn't have a `sceneIdentifier` class property like `StoryboardSceneBased`. Seems like an oversight since any view controller can have a storyboard ID.